controllers/contacts/contacts.js

```diff
@@ -0,0 +1,78 @@
+const contactsOperations = require("../../models/contacts");
+const { NotFound } = require("http-errors");
+
+const add = async (req, res, next) => {
+  const result = await contactsOperations.addContact(req.body);
+  res.status(201).json({
+    status: "success",
+    code: 201,
+    data: {
+      result,
+    },
+  });
+};
+
+const getAll = async (req, res) => {
+  const contacts = await contactsOperations.listContacts();
+  res.json({
+    status: "success",
+    code: 200,
+    data: {
+      results: contacts,
+    },
+  });
```

Comment:
```suggestion
      contacts 
``` 
аналогічно в інших місцях